source/agora/common/Serializer.d

```diff
@@ -119,21 +129,24 @@ public void serializePart (ubyte record, scope SerializeDg dg)
 public void serializePart (ushort record, scope SerializeDg dg)
     @trusted
 {
-    dg((cast(ubyte*)&record)[0 .. ushort.sizeof]);
+    dg(toVarInt(record));
```

Comment:
Since those are becoming trivial wrapper, use:
```D
public alias serializePart = toVarInt!(ushort);
```

Reply:
It doesn't work with overloads